### PR TITLE
Style/remember me checkbox

### PR DESCRIPTION
**Pull request by ayushkamadji**
Styling for #554 

Due to custom checkboxes requiring specific nesting of labels, this needs refactoring on session/new.html.erb with simple_form custom input & custom wrapper. I'll PR this to unblock other issues and work on it seperately.

### CODE REVIEW

**Comment by ayushkamadji** [comment]
I have the refactored version ready, but I'll wait on PR till this one is reviewed to see if any additional styling is needed.

File: app/assets/stylesheets/components/login_page.scss [hunk 1]
```diff
@@ -140,4 +140,37 @@
...
   }
+
+  .normal-checkbox {
+    display: none;
+    
+    + .custom-checkbox {
+      float: left;
+      margin-right: 3px;
+
+      > div {
```
Comment (KevinMulhern) [inline app/assets/stylesheets/components/login_page.scss]: Would be possible to apply these styles to the label instead?

File: app/assets/stylesheets/components/login_page.scss [hunk 2]
```diff
@@ -140,4 +140,37 @@
...
+        position: relative;
+        cursor: pointer;
+      }
+    }
+  }
+
+  .normal-checkbox:checked {
+    + .custom-checkbox > div:after {
+      content: '\f00c';
+      color: $black;
```
Comment (KevinMulhern) [inline app/assets/stylesheets/components/login_page.scss]: can we change this color to `$text-primary` please?

**Comment by KevinMulhern** [comment]
Can you explain the need for `CustomBooleanInput` please @ayushkamadji? 

This is looking really good 👍

**Comment by ayushkamadji** [comment]
The `CustomBooleanInput` essentially alters the output html of `BooleanInput`
from
```
<label>
  <input id="checkbox_1" type="checkbox">
  Label Text
</label>
```
to
```
<label>
  <input id="checkbox_1" type="checkbox">
  <label for="checkbox_1"></label>
  Label Text
</label>
```
I saw that writing a custom class allows for a cleaner html file, since doing this with default form input helpers leads to complicated nesting of helpers.

**Comment by KevinMulhern** [comment]
Makes sense, thanks @ayushkamadji 😄 
LGTM 👍

**Pull request merged**